packages/slice-machine/lib/builders/common/Zone/Card/components/ErrorTooltip.js

```diff
@@ -3,16 +3,16 @@ import { Fragment } from "react";
 import ReactTooltip from "react-tooltip";
 import { FaRegQuestionCircle } from "react-icons/fa";
 
-const ErrorTooltip = ({ errors }) => {
+const ErrorTooltip = ({ error }) => {
```

Comment:
would it be possible to migrate this to tsx?

Reply:
yes done